yugabytedb_managed/README.md

```diff
@@ -0,0 +1,72 @@
+# YugabyteDB Managed
+
+## Overview
+
+[YugabyteDB][1] is a cloud-native, distributed database that is API compatible with PostgreSQL.
+
+[YugabyteDB Managed][2] is YugabyteDB's fully-managed Database-as-a-service (DBaaS). Install YugabyteDB's integration to export metrics from a cluster into Datadog. The integration comes with an out-of-the-box dashboard to provide visibility into all the most important metrics needed to monitor the health and performance of a YugabyteDB Managed cluster such as:
+- Node resource usage (disk, memory, CPU, networking, and more).
+- Read/write operation throughput and latencies (both YSQL and YCQL).
+- Advanced Master and Tablet Server telemetry (for example, internal RPC throughput/latencies and WAL size).
+
+## Setup
+<div class="alert alert-warning">This feature is not available for <a href="https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-free/">Sandbox Clusters</a>.</div>
+<div class="alert alert-warning">For more information on setup, see the <a href="https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-monitor/metrics-export/#datadog">YugabyteDB documentation</a>.</div>
```

Comment:
I would leave this as regular text, this doesn't limit customer ability to use this integration. Add this link to the bottom of the page https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-monitor/metrics-export/#datadog
```suggestion
For more information on setup, see the [YugabyteDB documentation][x]
```